client/extensions/woocommerce/app/settings/shipping/shipping-zone/shipping-zone-locations.js

```diff
@@ -19,8 +19,8 @@ const ShippingZoneLocations = ( { translate } ) => {
 	return (
 		<div className="shipping-zone__locations">
 			<ExtendedHeader
-				label={ translate( 'Zone locations' ) }
-				description={ translate( 'Add locations that you want to share shipping methods' ) } >
+				label={ translate( 'Zone location' ) }
```

Comment:
Tricky one, a zone can obviously contain multiple locations. I wonder if making this label singular might suggest to folks that a zone can only contain a single location? I know the description says otherwise but people don't always read the descriptions xD